addons/stock/wizard/stock_inventory_already_set.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class StockInventoryAlreadySet(models.TransientModel):
```

Comment:
remove this wizard and instead call the dialog in js. You can check /mrp_mps/static/src/js/client_action.js _unlinkProduct

Reply:
ignore that at the moment